determined-ai/determined
# fix: avoid flashing previous project experiment list when navigating to uncategorized experiments [WEB-908]

## Description

https://determinedai.atlassian.net/browse/WEB-908

add key to ProjectDetails routes





## Test Plan

- Navigate to Uncategorized Experiments and confirm that there are experiments showing up there that are uncategorized
- Go create a workspace if you don’t already have one
- Go create a new project within that workspace (can be named anything)
- Navigate back to Uncategorized Experiments and you should see the uncategorized experiments (it should NOT show the previous projects list, which is empty since it's newly created)





## Commentary (optional)





## Checklist

  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.

## Ticket
